migrate.sh

```diff
@@ -2,11 +2,4 @@
 
 cd /rails_app
 
-ln -sf /rails_conf/* ./config/
-
-if [ -z "$RAILS_ENV" ]
-then
-    export RAILS_ENV="production"
```

Comment:
I consider this dangerous to leave laying around, it will default to development to avoid someone running this and accidentally running against production db.